src/test/msixtest/unbundle.cpp

```diff
@@ -56,7 +56,7 @@ TEST_CASE("Unbundle_StoreSigned_Desktop_x86_x64_MoviesTV", "[unbundle]")
     CHECK(MsixTest::Directory::CompareDirectory(outputDir, files));
 
     // Verify non expected files aren't present
-    auto filesNotApplicable = MsixTest::Unbundle::GetExpectedFilesNoApplicable();
+    auto filesNotApplicable = MsixTest::Unbundle::GetExpectedFilesNoLanguageApplicable();

```

Comment:
>GetExpectedFilesNoLanguageApplicable [](start = 50, length = 36)

Why is this one NoLanguageApplicable? Shouldn't unbundle with full applicability also avoid unpacking scale?

Reply:
You're right, thanks for catching this!